source/manual/assets.html.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: GOV.UK assets
+title: Assets: how they work
```

Comment:
This title needs to be in quotes because of the colon - it raises an error that the page doesn't have a title otherwise.